JulioC-19/LargeProjectMobile
# Julio/recipe screen

Implemented recipe screen, top bar navigator for favorites/pantry and md5 password for hashing. 

To test: 
Log in with the following credentials:
pablo@email.com
pablo
Navigate to home screen and tab a recipe, a modal should pop up with the recipe details, ingredients are not implemented yet.
Navigate to favorites and test the top nav bar, it should move between favorite screen and pantry screen.
